app/elements/pw-shell.html

```diff
@@ -250,13 +286,53 @@
 
       // Save the initial URL so that we don't double-render the first load.
       this._currentPageURL = this._sectionRoute.path;
+
+      // Save which version of docs we're looking at, since we need to
+      // maintain this state across the app.
+      this.updateDocsVersion();
+
+      // Update the version when the version selector is used.
+      this.$.versionSelect.addEventListener('iron-select', function(event) {
+        var newSelection = event.detail.item.textContent;
+        if (newSelection === this.activeDocsVersion)
+          return;
+        this.activeDocsVersion = newSelection;
+        window.location = '/' + this.activeDocsVersion + '/start/';
```

Comment:
In the current architecture, there's no data flow from devguide-shell to pw-shell, so setting  this._sectionRoute.path from the dropdown in devguide-shell wouldn't work. Instead, this could be made more similar to how other normal `<a>` links work - iron-location installs a global click handler which prevents default on link clicks, does a `window.history.pushState` with the new path, and fires 'location-changed' on window. So on select changed, we can do a similar thing thru `window`:

```javascript
window.history.pushState({}, '', path);
this.fire('location-changed', {}, {node: window});
```